Carmunication/Views/SessionStart/SessionStartMidNoGroupView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SessionStartMidNoGroupView.swift
+//  Carmunication
+//
+//  Created by 김태형 on 2023/10/18.
+//
+
+import UIKit
+
+import SnapKit
+
+final class SessionStartMidNoGroupView: UIView {
+
+    private lazy var logoImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(systemName: "heart")
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+    private lazy var firstCommentLabel: UILabel = {
+        let lbl = UILabel()
+        lbl.textColor = UIColor.semantic.textPrimary
+        lbl.font = UIFont.carmuFont.headline1
+        lbl.textAlignment = .center
+        lbl.numberOfLines = 0
+        let fullText = "초대하거나 받은 여정이 없어요\n친구와 함께 여정을 시작해보세요!"
+        let attributedText = NSMutableAttributedString(string: fullText)
+        let range = (fullText as NSString).range(of: "여정을 시작")
+        attributedText.addAttributes([.foregroundColor: UIColor.semantic.accPrimary as Any], range: range)
+        lbl.attributedText = attributedText
+        return lbl
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupUI()
+        setupConstraints()
+    }
```

Comment:
required init?은 스토리보드나 xib을 사용할 때 호출되는 initializer라고 하더라구요
저희는 사용하지 않으니 아래처럼 수정해도 될 것 같아요 :)

```suggestion
    required init?(coder: NSCoder) {
        super.init(coder: coder)
    }
```

Reply:
그럼 setupUI()와 setupConstraints()를 호출하는 것은 어디에서 호출하였나요?!
해당 문제는 모든 팀원들과 공유를 해야될 거 같은데, 모든 팀원들이 아마 여기에서 실행하는 것으로 알고 있습니다,,!

Reply:
setupUI()와 setupConstraints()는 작성해주신 initializer `override init(frame:)`에서 이미 호출되고 있어요!
이니셜라이저들이 어떤 경우에 호출되는지 알아보시는 것도 좋을 것 같네요

https://velog.io/@dev_jane/UIViewController-%EC%84%9C%EB%B8%8C%ED%81%B4%EB%9E%98%EC%8A%A4%EC%9D%98-custom-initializer-%EB%A7%8C%EB%93%A4%EA%B8%B0required-initializer-initcoder-must-be-provided-by-subclass-of-UIViewController

```swift
    override init(frame: CGRect) {
        super.init(frame: frame)
        setupUI()
        setupConstraints()
    }
```

Reply:
넵넵 우선 구현에 집중하다보니 놓치는 부분이 있었네여,, 🥲 꼭 읽어보겠습니당 !!